+Prefs/+Inputs/BooleanField.m

```diff
@@ -3,14 +3,32 @@
 
     properties(Hidden)
         uistyle = 'checkbox';
+        nanuistyle = 'radiobutton';
```

Comment:
Changes to Prefs.Boolean to allow_nan. NaN is displayed as a circular (and readonly) radiobutton to distinguish it from the square checkbox on or checkbox off.